src/components/UserProfile/AddNewUserProfile/UserProfileAdd.jsx

```diff
@@ -244,7 +244,10 @@ class AddUserProfile extends Component {
                     <FormGroup>
                       <TimeZoneDropDown
                       filter={this.state.timeZoneFilter}
-                      onChange={this.handleUserProfile}
+                      onChange={(e) => {
+                        this.handleUserProfile(e);
+                        console.log('@@@@@@@@@@@@@@@@@@@@@@@@@@');
```

Comment:
Remove console.log?

Reply:
Fixed this issue and added a couple of changes I forgot to add originally.

Reply:
That's fine but there is one conflict. I believe it is because of the tiny addition of the endDate portion in UserProfile.js.